lib/phoenix_mtm/changeset.ex

```diff
@@ -41,6 +41,7 @@ defmodule PhoenixMTM.Changeset do
   end
 
   defp all(ids, repo, mod) do
-    repo.all(from m in mod, where: m.id in ^ids)
+    ids_as_ints = Enum.map(ids, fn(i) -> if String.length(i) > 0, do: String.to_integer(i) end)
```

Comment:
My concern with this is that perhaps `id` isn't always an integer, but a binary UUID or even a string on purpose (ie. slug).

What was your use case here? I'd almost think this should be done in your controller.